.github/workflows/ios-ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: macos-10.15
+
+    steps:
+    # Checkout the repo
+    - uses: actions/checkout@v2
```

Comment:
What does this mean? Like are there docs I can view? It's all good but I'm just trying to wrap my head around how GH Actions works

Reply:
https://github.com/actions/checkout#checkout-v2

Reply:
Thanks